Amethyst/Preferences/LayoutsPreferencesViewController.swift

```diff
@@ -12,6 +12,7 @@ class LayoutsPreferencesViewController: NSViewController, NSTableViewDataSource,
     private var layoutKeys: [String] = []
 
     @IBOutlet var layoutsTableView: NSTableView?
+    @IBOutlet weak var relaunchBtn: NSButton?
```

Comment:
It looks like the xib connects to `relaunchButton`, not `relaunchBtn`. I couldn't get the UI to work without renaming it to the former.